### ISSUE TITLE
Restricting the size of MySQL query 

### ISSUE THREAD

**Issue by srikanthBezawada** [packetbeat/packetbeat#96]
Hi,

Is there any way to restrict the size of MySQL query while sending to elastic search ?

Thanks,
Srikanth.

**Comment by tsg** [comment]
No, sorry, we have some flexibility for that in the HTTP code, but not yet for Mysql. We're doing a code reorganisation in preparation for 0.5.0, which will make it possible to implement these kind of things once and have them automatically for all protocols.

**Comment by ibrahimsharaf** [comment]
Hi @monicasarbu, I want to start working on this one, how can I start?

**Comment by monicasarbu** [comment]
@ibrahimsharaf Great to hear you would like to start working on this one. I do something similar in Packetbeat for the HTTP protocol: https://github.com/elastic/beats/blob/master/packetbeat/protos/http/http.go#L603

**Comment by ibrahimsharaf** [comment]
Hi @monicasarbu, sorry for the delay, would the changes take place in this file (https://github.com/elastic/beats/blob/master/packetbeat/protos/mysql/mysql.go)? is the change about restricting the size of the query in terms of query length?

**Comment by ak1t0** [comment]
Hi @monicasarbu. I would like to work on this issue.

**Comment by ruflin** [comment]
@ak1t0 Great to hear. Feel free to open a PR as soon as you feel comfortable to discuss the code.

**Comment by onlinejudge95** [comment]
@ruflin Any updates on this? I checked the PR associated with this. closed down due to inactivity. I am interested to take it further

**Comment by jlind23** [comment]
Backlog grooming: Stalled issue, closing for now. Feel free to reopen it if needed

**Comment by ak1t0** [comment]
Sorry for very late reply. 
At the time I did not understand anything about this project and issue.
I apologize for the inconvenience and its cost. 
Please close this issue.

:bowing_man:  :bowing_man:  :bowing_man:
